lib/Runtime/Library/RegexHelper.cpp

```diff
@@ -1718,4 +1818,12 @@ namespace Js
         Var result = RegexHelper::RegexSplitImpl(entryFunctionContext, regularExpression, input, limit, noResult, stackAllocationPointer);
         return RegexHelper::CheckCrossContextAndMarshalResult(result, entryFunctionContext);
     }
+
+    int64 RegexHelper::AdvanceStringIndex(JavascriptString* string, int64 index, bool isUnicode)
```

Comment:
It'll be used to determine the size of the code point at `index`. I'm having all callers pass it so that  we'll only change this function when needed.

Just curious... Would the VS C++ compiler put an unused argument on the stack or would it actually eliminate the argument? @abchatra, do you know?